lib/p2p/Pool.ts

```diff
@@ -366,6 +366,15 @@ class Pool extends EventEmitter {
     return false;
   }
 
+  /**
+   * Current network information that comes from xud configuration
+   *
+   * @return returns currently configured network
```

Comment:
This shouldn't need "return" twice so just `@returns the active xu network` or similar.